src/dev/flang/fuir/analysis/dfa/DFA.java

```diff
@@ -915,7 +915,7 @@ static Value NYIintrinsicMissing(Call cl)
     put("fuzion.std.exit"                , cl -> null );
     put("fuzion.std.out.write"           , cl -> Value.UNIT );
     put("fuzion.std.err.write"           , cl -> Value.UNIT );
-    put("fuzion.std.fileio.read"         , cl -> NYIintrinsicMissing(cl) );
+    put("fuzion.std.fileio.read"         , cl -> Value.UNIT );
```

Comment:
please add a comment `NYI: need to record modification of array passed as argument` or similar.